blueprints/load-scheduling/common/utils.libsonnet

```diff
@@ -39,13 +39,7 @@ local add_kubelet_overload_confirmations(c) = {
   local updated_cfg = c {
     policy+: {
       service_protection_core+: {
-        local overloadConfirmations = if std.objectHas(c.policy.service_protection_core, 'overload_confirmations') then
```

Comment:
Why did we remove this? This was added for users who wants to use some other signals than CPU and memory usage and have provided the same via the values file. With this change, even if user provides some values under `overload_confirmations`, that would get skipped

Reply:
With this line, `overload_confirmations` where being added twice, in turn, causing the generated policy to have double the amount of components.